tools/walletextension/multi_acc_helper_test.go

```diff
@@ -58,8 +58,8 @@ func TestErrorsWhenDataFieldIsMissing(t *testing.T) {
 func TestGracefulWhenDataFieldTooShort(t *testing.T) {
```

Comment:
@otherview The test name needs to change - it's the opposite of graceful now.

Reply:
Fixed 👍